### PR TITLE
feat/0x35 calldataload opcode (Test Refactor)

### PR DESCRIPTION
**Pull request by TotalPizza**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #101 

## What is the new behavior?

- CALLDATALOAD opcode
- Reading bytes (variable size) from calldata
- test coverage

### CODE REVIEW

**Comment by TotalPizza** [comment]
Seeing if checks are re-enabled.

**Comment by TotalPizza** [comment]
I have some issues with the local poetry installation.
Some dependencies aren't working that should be, the usual.
Will probably get that fixed tomorrow and ensure that files are formatted correctly.

**Comment by LucasLvy** [comment]
Use the rule format-mac

File: src/kakarot/instructions/environmental_information.cairo [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 // Starkware dependencies
 
+from starkware.cairo.common.alloc import alloc
```
Comment (LucasLvy) [inline src/kakarot/instructions/environmental_information.cairo:7]:
```suggestion
```
Alloc is already imported below

**Review by LucasLvy — CHANGES_REQUESTED** [review]

File: src/kakarot/instructions/environmental_information.cairo (hunk 1 above)
Comment (TotalPizza) [reply src/kakarot/instructions/environmental_information.cairo:7]: Ah, saw this a bit late. Commited that fix just now :)

**Review by abdelhamidbakhta — APPROVED** [review]
lgtm

**Pull request merged**